client/src/screens/groups/GroupDetails.js

```diff
@@ -325,36 +138,72 @@ class GroupDetails extends Component {
 
     // somehow we got here with no group
     if (!group) {
-      return <Text>Error displaying group</Text>;
+      return (
+        <Container>
+          <Center>
+            <Text>Nothing here. thats weird.</Text>
+          </Center>
+        </Container>
+      );
     }
 
-    const rows = [
-      ['info', group],
-    ].concat(
-      _.map(['Users'], r => ['section', r]),
-      _.map(group.users, r => ['user', r]),
-      _.map(['Events'], r => ['section', r]),
-      _.map(group.events, r => ['event', r]),
-      _.map(['Schedules'], r => ['section', r]),
-      _.map(group.schedules, r => ['schedule', r]),
-    );
-    // render group stuff
-
     const memberAlready = _.some(group.users, g => g.id === me.id);
+    const tags = group.tags.map(elem => `#${elem.name}`).join(', ');
```

Comment:
we could put this in a utils file or something so we can reuse it across the app

Reply:
agree. theres a few places that use it so ill open a new PR